File: Completo/ws21GUI/ws21GUISwing.py/ShoppingCenter/controller/user_controller.py
from model.user import User

class UserController:
    def __init__(self):
        self.user = None

    def login(self, user_name, password):
        self.user = User(1, "Christopher", "Christopher")
        return self.user.get_user_name() == user_name and self.user.get_password() == password

File: Completo/ws21GUI/ws21GUISwing.Java/ws21GUISwing/ShoppingCenter/src/main/java/ec/edu/espe/shoppingcenter/controller/UserController.java

package ec.edu.espe.shoppingcenter.controller;

import ec.edu.espe.shoppingcenter.model.User;

/**
 *
 * @author Christopher Lomas,Polimorphismus ,@ESPE
 */
public class UserController {
    User user;
    
    public boolean login(String userName, String password){
        
        user= new User(1,"Christopher","Christopher");
    
        return user.getUserName().equals(userName) && user.getPassword().equals(password);
    
    }
    
}
